tests/unit/admin/test-class-wc-rest-payments-survey-controller.php

```diff
@@ -63,33 +63,42 @@ public function test_empty_rating_returns_400_status_code() {
 		$this->assertEquals( 400, $response->get_status() );
 	}
 
+
 	public function test_valid_request_forwards_data_to_jetpack() {
```

Comment:
How important and useful is this e2e test, considering that the survey has a limited lifespan?